### PR TITLE
Improved the Workplace Search documentation

### PR DESCRIPTION
**Pull request by ezimuel**
Added a search example in the Workplace Search documentation

### CODE REVIEW

File: docs/guide/workplace-search-api.asciidoc [hunk 1]
```diff
@@ -28,7 +34,14 @@ $result = $workplace->createContentSource(
 $sourceId = $result['id'] ?? null;
 ----------------------------
 
-=== Documents
+We need to store the `sourceId` since it's required to perform other operations, like index a document.
+
+=== Index documents
+
+To store a document in Workplace Search you need to perform an index operation.
+YOu can add one or more document at time.
```
Comment (ppf2) [inline docs/guide/workplace-search-api.asciidoc]: YOu -> You :)

**Comment by ppf2** [comment]
Thx! LGTM (I haven't tested the code).  I like that we are providing an example from creating a custom content source to indexing, searching and deleting the document.  

Perhaps we can also provide an example of how users can fetch an existing first party content source back in addition to creating a custom content source?

File: docs/guide/workplace-search-api.asciidoc (hunk 1 above)
Comment (ezimuel) [reply docs/guide/workplace-search-api.asciidoc]: Fixed

**Comment by ezimuel** [comment]
@ppf2 the search API will return all the documents, including the ones from the third-party services. The example for the custom content source is only for adding some documents in the archive.

**Pull request merged**